SmartCmdArgs/SmartCmdArgs17/SmartCmdArgs17.csproj

```diff
@@ -76,7 +76,10 @@
       <Version>2.0.6142705</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.0.31902.203" ExcludeAssets="runtime" />
-    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.0.5233" />
+    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.5.4065">
```

Comment:
This updated because I had to build for Windows 11 Arm64, because I'm developing on an M1 Mac w/ Parallels.